tmva/tmva/test/DNN/CNN/TestConvNetPrediction.cxx

```diff
@@ -0,0 +1,75 @@
+// @(#)root/tmva $Id$
+// Author: Vladimir Ilievski, 16/06/17
+
+/*************************************************************************
+ * Copyright (C) 2017, Vladimir Ilievski
```

Comment:
Please change all copyrights to the ones used in ROOT. Otherwise I do not think we can accept this contribution.